app/src/main/java/org/sopt/and/SignUpActivity.kt

```diff
@@ -0,0 +1,213 @@
+                    style = MaterialTheme.typography.titleLarge.copy(lineHeight = 32.sp),
+                    color = Color.White, fontWeight = FontWeight.Normal,
+                )
+
+                InputField("wavve@email.com", email, {email = it})
+                Text(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 8.dp),
+                    text = "로그인, 비밀번호 찾기, 알림에 사용되니 정확한 이메일을 입력해 주세요.",
+                    style = MaterialTheme.typography.bodySmall, color = Color.Gray,
+                    fontWeight = FontWeight.Normal
+                )
+
+                PasswordInputField ("SoptAnd123@@", password, {password = it}, passwordVisible, {passwordVisible = !passwordVisible},)
+                Text(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 8.dp),
+                    text = "비밀번호는 8~20자 이내로 영문 대소문자, 숫자, 특수문자 중 3가지 이상 혼용하여 입력해 주세요.",
+                    style = MaterialTheme.typography.bodySmall, color = Color.Gray,
+                    fontWeight = FontWeight.Normal
+                )
+
+                Spacer(modifier = Modifier.weight(0.5f))
+
+                Box(
+                    modifier = Modifier
+                    .fillMaxWidth(),
+                    contentAlignment = Alignment.Center
+                ){
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically
+                    ){
+                        Spacer(modifier = Modifier.weight(1f).height(1.dp).background(Color.Gray))
+                        Text("또는 다른 서비스 계정으로 가입", modifier = Modifier.padding(horizontal = 8.dp), style = MaterialTheme.typography.bodySmall, fontWeight = FontWeight.Normal)
+                        Spacer(modifier = Modifier.weight(1f).height(1.dp).background(Color.Gray))
+                    }
+                }
+                SocialLoginRow()
+                Text(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(8.dp),
+                    text = "SNS계정으로 간편하게 기입하여 서비스를 이용하실 수 있습니다. 기존 POOQ 계정 또는 Wavve 계정과는 연동되지 않으니 이용에 참고하세요",
+                    style = MaterialTheme.typography.bodySmall.copy(lineHeight = 20.sp),
+                    color = Color.Gray, fontWeight = FontWeight.Normal,
+                )
+
+                Spacer(modifier = Modifier.weight(1f))
+                Button(
+                    onClick = { onSignUpClick() },
+                    modifier = Modifier.fillMaxWidth(),
+                    colors = ButtonDefaults.buttonColors(Color.LightGray),
+                    shape = RectangleShape
+                ){
+                    Text("Wavve 회원가입")
+                }
+            }
+        }
+    )
+}
+
+
+private fun isValidEmail(email: String): Boolean {
+    return android.util.Patterns.EMAIL_ADDRESS.matcher(email).matches()
+}
```

Comment:
```suggestion
private fun isValidEmail(email: String): Boolean = android.util.Patterns.EMAIL_ADDRESS.matcher(email).matches()
```
return만 있는 함수는 이렇게 작성할 수도 있ㅇㅓ요!